### ISSUE TITLE
Add option for including release notes in update dialog?

### ISSUE THREAD

**Issue by davej** [electron/update-electron-app#13]
Just wondering if showing the release notes when prompting the user to restart is on the roadmap? Release notes would be taken from the github release. 

Here's a potential mock-up:

![release-notes](https://user-images.githubusercontent.com/26971/39094673-64c6fc40-462b-11e8-9ec2-76deef24e73e.png)

An option could toggle whether to show them or not:

```js
require('update-electron-app')({
  repo: 'ummoji/ummoji-desktop',
  releaseNotes: true
})
```

Even if release notes are turned on they would only be shown if there are actually release notes with the release. The release notes would probably need to be contained with a maximum height and overflow content should scroll because otherwise the height of the dialog may be too large. Perhaps this should be configurable or perhaps not.

**Comment by zeke** [comment]
@davej I'm in favor of this. Do you want to give it a try?

**Comment by zeke** [comment]
> Or perhaps there's a way to approach it without touching Electron core?

Yeah that would be ideal. Since this is designed for use with GitHub Release, we could also offer the user a link to the release page... 🤔 

@MarshallOfSound @zcbenz is there a way to know at app startup that the app itself was recently updated? Don't see a hook for that in the autoUpdater docs.

**Comment by zcbenz** [comment]
> @MarshallOfSound @zcbenz is there a way to know at app startup that the app itself was recently updated? Don't see a hook for that in the autoUpdater docs.

The native auto-update frameworks used in Electron do not provide the functionality, it is usually implemented in apps by storing current version in a configuration file.

**Comment by quanganh1906** [comment]
cảm ơn rất nhiều

**Comment by erickzhao** [comment]
Ref: https://github.com/electron/electron/pull/23692

Not exactly the toggle, but definitely and improved experience

**Comment by jhaemin** [comment]
No more updates?
